client/views/chat.tpl

```diff
@@ -1,32 +1,32 @@
 {{#each channels}}
 <div id="chan-{{id}}" data-title="{{name}}" data-id="{{id}}" data-type="{{type}}" data-target="#chan-{{id}}" class="chan {{type}}">
 	<div class="header">
-		<button class="lt" aria-label="Toggle channel list"></button>
+		<button class="lt" aria-label="{{translate 'client.toggle_chan_list'}}"></button>
 		{{#equal type "channel"}}
-			<span class="rt-tooltip tooltipped tooltipped-w" aria-label="Toggle user list">
-				<button class="rt" aria-label="Toggle user list"></button>
+			<span class="rt-tooltip tooltipped tooltipped-w" aria-label="{{translate 'client.toggle_user_list'}}">
+				<button class="rt" aria-label="{{translate 'client.toggle_user_list'}}"></button>
 			</span>
 		{{/equal}}
-		<button class="menu" aria-label="Open the context menu"></button>
+		<button class="menu" aria-label="{{translate 'open_context_menu'}}"></button>
 		<span class="title">{{name}}</span>
 		<span title="{{topic}}" class="topic">{{{parse topic}}}</span>
 	</div>
 	<div class="chat">
 		<div class="show-more {{#equal messages.length 100}}show{{/equal}}">
-			<button class="show-more-button" data-id="{{id}}">Show older messages</button>
+			<button class="show-more-button" data-id="{{id}}">{{translate 'client.show_older_messages'}}</button>
 		</div>
 		<div class="messages"></div>
 	</div>
 	{{#equal type "channel"}}
 	<aside class="sidebar">
 		<div class="users">
 			<div class="count">
-				<input type="search" class="search" aria-label="Search among the user list" tabindex="-1">
+				<input type="search" class="search" aria-label="{{translate 'client.search_among_user_list'}}" tabindex="-1">
```

Comment:
I'm not sure where yet, but this is getting a `placeholder="{{amount}} users"`, the 'users' isn't translated

Reply:
This and the comment below are done in javascript. nice catch